### ISSUE TITLE
slcli file volume-detail throws an exception

### ISSUE THREAD

**Issue by alsuarez** [softlayer/softlayer-python#813]
```
$ slcli file volume-detail <id>
An unexpected error has occured:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/SoftLayer/CLI/core.py", line 163, in main
    cli.main(**kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/SoftLayer/CLI/file/detail.py", line 19, in cli
    used_space = int(file_volume['bytesUsed'])
TypeError: int() argument must be a string or a number, not 'NestedDict'
```
### Expected Behavior
The file storage details are returned
### Actual Behavior
An exception is thrown
### Environment Information
Operating System: OSX 10.11.6
softlayer-python version (`slcli --version`): 5.2.4

**Comment by alsuarez** [comment]
The file storage I was wanting to get details from was brand new. the `file_volume` dict didn't have a `bytesUsed` key in it.

**Comment by allmightyspiff** [comment]
yeah i figured that was the issue, but I would still like the client to fail gracefully, so i'll keep this issue open until I can get it resolved.

**Comment by miravck** [comment]
I ran into the same issue with SoftLayer 5.2.5 on a new file storage volume.

`
An unexpected error has occured:
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/SoftLayer/CLI/core.py", line 163, in main
    cli.main(**kwargs)
  File "/Library/Python/2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Library/Python/2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Python/2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Python/2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Python/2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/Library/Python/2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/SoftLayer/CLI/file/detail.py", line 19, in cli
    used_space = int(file_volume['bytesUsed'])
TypeError: int() argument must be a string or a number, not 'NestedDict'
`

**Comment by allmightyspiff** [comment]
This was fixed in #830
